### PR TITLE
2841 update args

### LINKED ISSUE #2841: Investigating an unexpected behaviour reported for the `enable indirect` API endpoint
Following up from this [community post](https://community.opentargets.org/t/spurious-indirect-association-evidence-via-graphql-api/879), we plan to investigate and fix the problem causing an unexpected behaviour for the `enable indirect` API endpoint. 
This endpoint is not exposed in the frontend, and therefore not extensively tested.

<details>
 <summary>This example API query provided by the user.</summary>

```

query associatedDiseases {
  target(ensemblId: "ENSG00000160710") {
    id
    approvedSymbol
    associatedDiseases(enableIndirect: true) {
      count
      rows {
        score
        disease {
          id
          name
        }
        datasourceScores {
          id
          score
        }
        datatypeScores {
          id
          score
        }
      }
    }
  }
}
The output:

{
  "data": {
    "target": {
      "id": "ENSG00000160710",
      "approvedSymbol": "ADAR",
      "associatedDiseases": {
        "count": 11197,
        "rows": [
          {
            "score": 0.915009673664289,
            "disease": {
              "id": "EFO_0000222",
              "name": "acute myeloid leukemia"
            },
            "datasourceScores": [
              {
                "id": "chembl",
                "score": 0.9920171593572428
              },
[...]
            ],
            "datatypeScores": [
              {
                "id": "known_drug",

```

</details>

------

First observations suggesting this is a backend problem:

1) From the example reported by the user, there is no chEMBL evidence displayed on the [evidence page](https://platform.opentargets.org/evidence/ENSG00000160710/EFO_0000222)

1) BigQuery on the `associationByDatasourceIndirect` table does not return any indirect evidence neither 
<details>
 <summary>BigQuery response</summary>

```

SELECT * FROM `open-targets-eu-dev.platform_dev.associationByDatasourceIndirect` WHERE targetId = "ENSG00000160710" AND diseaseId = "EFO_0000222" LIMIT 10

datatypeId	datasourceId	diseaseId	targetId	score	evidenceCount
literature	europepmc	EFO_0000222	ENSG00000160710	0.656767905	6

```
</details>

### PR DESCRIPTION
**Pull request by remo87**
This item starts the process to fix the issues related to [2841](https://github.com/opentargets/issues/issues/2841) by adding descriptions to the arguments for the associations on the flight query arguments
- Add argument to indirect Update arguments descriptions add note on unused arg
- formatting files

### CODE REVIEW

File: app/models/gql/Arguments.scala [hunk 1]
```diff
@@ -86,14 +97,22 @@ object Arguments {
...
+    OptionInputType(StringType),
+    description = "Ordering for the associations. By default is score desc"
+  )
 
   val AId: Argument[String] = Argument("A", StringType)
   val AIds: Argument[Seq[String with tag.Tagged[FromInput.CoercedScalaResult]]] =
     Argument("As", ListInputType(StringType))
   val BIds: Argument[Option[Seq[String]]] =
-    Argument("Bs", OptionInputType(ListInputType(StringType)))
+    Argument("Bs", OptionInputType(ListInputType(StringType)), description = "List of disease IDs")
```
Comment (jdhayhurst) [inline app/models/gql/Arguments.scala]: "Bs" can also be target IDs, it depending on whether you're on a target or disease page.

**Review by jdhayhurst — COMMENTED** [review]
Nice! just one thing (see the comment)

File: app/models/gql/Arguments.scala (hunk 1 above)
Comment (remo87) [reply app/models/gql/Arguments.scala]: True thanks @jdhayhurst I'll apply the change

Comment (remo87) [reply app/models/gql/Arguments.scala]: how do you guys feel about "List of disease or target IDs"

Comment (jdhayhurst) [reply app/models/gql/Arguments.scala]: yes, I think that's probably ok

**Review by jdhayhurst — APPROVED** [review]

**Pull request merged**